mavis/test/models.py

```diff
@@ -130,6 +130,19 @@ class Vaccine(StrEnum):
     REVAXIS = "Revaxis"
 
 
+class VaccinationSite(StrEnum):
```

Comment:
```suggestion
class DeliverySite(StrEnum):
```

In the service it's called `DeliverySite` so it might be worth being consistent in the tests, but not a big deal.